### ISSUE TITLE
K810 Keyboard Fn Keys Issue

### ISSUE THREAD

**Issue by szvry** [tekezo/Karabiner-Elements#627]
Hey. I use karabiner elements to remap keyboard windows keys to work like on Mac Keyboard. Unfortunately enabling karabiner also change my Function keys. Before karabiner I could use Function keys without pressing FN button to make them work (increase, decrease music, play/pause). Right now with Karabiner on I have to press FN to use function keys. Karabiner EventViewer doesn't even see Function keys(excluding F13 which is print option on windows), without using Fn button, doesn't see Fn key itself too. 

Is there any solution for that?

**Comment by tyrocca** [comment]
I have been having a similar thing. If I use my laptop's fn key things work. But the fn key on my Razer Chroma doesn't do anything. If I turn of karabiner then it works normally.

**Comment by billputer** [comment]
I was having a similar problem. I want my internal Macbook Pro keyboard function keys to do the normal OS X functions (volume up and down, etc), and I want my external keyboard function keys to do the normal F1, F2, etc. This was the default for Karabiner, and is the default without Karabiner Elements installed.

However, I just installed 0.91.13, which allows you to configure the function keys on a per-device basis. I've tested, and it definitely fixes things for me.
